app/assets/javascripts/3dbio_viewer/src/domain/entities/UploadData.ts

```diff
@@ -0,0 +1,121 @@
+export interface UploadData {
+    dataUrl: string;
+    chains: Chains;
+}
+
+export interface Chains {
+    A: ChainObject[];
+    B: ChainObject[];
+    C: ChainObject[];
+}
```

Comment:
As chains can be "any" string, we don't need to worry about typing it: `Record<string, ChainObject[]>`